flux-middle/src/rty/mod.rs

```diff
@@ -194,6 +194,7 @@ pub enum BaseTy {
     Slice(Ty),
     Adt(AdtDef, Substs),
     Float(FloatTy),
+    Ptr(Ty, Mutability),
```

Comment:
We should call this `RawPtr` to differentiate from `rty::TyKind::Ptr`

```suggestion
    RawPtr(Ty, Mutability),
```